src/reporters/CLIReporter.ts

```diff
@@ -77,6 +85,7 @@ export class CLIReporter implements IReporter {
     lines.push(italic(blue('We can help you with both. :-)')));
     lines.push(italic(blue('- https://dxheroes.io')));
     lines.push(reset(' '));
+    lines.push(grey(`You can run the command with option ${italic('-d')} or ${italic('--details')} to show detailed informations.`));
```

Comment:
Does it show also if the user already used the flag?

Reply:
good point! fixed